### PR TITLE
realtime: add raspi variant

### PR DESCRIPTION
**Pull request by orndorffgrant**
## Why is this needed?

A raspberry pi variant of realtime-kernel may become available in time for noble. This sets up the variant on our side.

## Test Steps

run the edited lxd-vm behave test

## Checklist

 - [x] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] Changes here need to be documented, and this was done in: 

## Does this PR require extra reviews?

 - [x] Yes from realtime-team
 - [ ] No

### CODE REVIEW

File: uaclient/messages/__init__.py [hunk 1]
```diff
@@ -1426,6 +1426,8 @@ class TxtColor:
 REALTIME_NVIDIA_DESCRIPTION = t.gettext(
     "RT kernel optimized for NVIDIA Tegra platform"
 )
+REALTIME_RASPI_TITLE = t.gettext("Real-time Raspberry Pi Kernel")
+REALTIME_RASPI_DESCRIPTION = t.gettext("RT kernel optimized for Raspberry Pi")
```
Comment (orndorffgrant) [inline uaclient/messages/__init__.py:1430]: @EdoardoBarbieriCanonical Are these a reasonable title and description of the raspberry pi variant? Should we mention a specific version or versions of the rpi?

Also, I have named the variant "raspi", so the command would be `pro enable realtime-kernel --variant=raspi`.

We need to decide all of these things now if we want this to even be a possibility in April.

Comment (EdoardoBarbieriCanonical) [reply uaclient/messages/__init__.py:1430]: Thanks for the PR. Can we go with?

- Title: "Raspberry Pi Real-time for Pi5/Pi4"
- Description: "24.04 Real-time kernel optimised for Raspberry Pi"
- Command: `pro enable realtime-kernel --variant=rpi`

Comment (orndorffgrant) [reply uaclient/messages/__init__.py]: Added. Thanks @EdoardoBarbieriCanonical!

File: features/realtime_kernel.feature [hunk 2]
```diff
@@ -299,7 +316,8 @@ Feature: Enable command behaviour when attached to an Ubuntu Pro subscription
         realtime-kernel  yes +disabled   +Ubuntu kernel with PREEMPT_RT patches integrated
         ├ generic        yes +disabled  +Generic version of the RT kernel \(default\)
         ├ intel-iotg     yes +disabled  +RT kernel optimized for Intel IOTG platform
-        └ nvidia-tegra   yes +disabled   +RT kernel optimized for NVIDIA Tegra platform
+        ├ nvidia-tegra   yes +disabled  +RT kernel optimized for NVIDIA Tegra platform
+        └ rpi            yes +disabled  +RT kernel optimized for Raspberry Pi
```
[the remark this answers is not in the document]
Comment (orndorffgrant) [reply features/realtime_kernel.feature]: updated! and CI looks good now

**Review by lucasmoura — APPROVED** [review]

**Review by renanrodrigo — APPROVED** [review]

**Pull request merged**